deps/src/polymake_functions.h

```diff
@@ -34,4 +34,10 @@ struct VecOrMat_supported
     typedef jlcxx::ParameterList<pm::Int, pm::Integer, pm::Rational, double> value_type;
 };
 
+struct Polynomial_supported
+{
+    typedef jlcxx::ParameterList<int, pm::Integer, pm::Rational, double,
+        pm::Polynomial<int,int>, pm::Polynomial<pm::Integer,int>, pm::Polynomial<pm::Rational,int>, pm::Polynomial<double,int>> value_type;
+};
+
```

Comment:
Is this ever used?

Reply:
you are right, it is not. for now, we can restrict to the types used in the context of vectors or matrices. if we should ever consider expanding the support, i.e. polymake supports polynomials of polynomials, then something like this would be needed. so i just removed it from here